tests/unit/test-firefox/test.firefox.js

```diff
@@ -629,14 +629,13 @@ describe('firefox', () => {
       const fakeProfiles = [{Name: 'someName'}];
       const userDirectoryPath = '/non/existent/path';
 
-      const FxProfile = {
-        Finder() {
-          return {
-            readProfiles: fakeReadProfiles,
-            getPath: fakeGetPath,
-            profiles: fakeProfiles,
-          };
-        },
+      const FxProfile = {};
+      FxProfile.Finder = function Finder() {
```

Comment:
Why this change?

Reply:
These tests has started to fail because of Finder wasn't considered a constructor, I think that the upgrade from babel 6 to babel 7 somehow unveiled the issue.